packages/components/src/shortcuts/Shortcut.ts

```diff
@@ -0,0 +1,324 @@
+/* eslint-disable no-underscore-dangle */
+import React from 'react';
+import { ContextActionUtils } from '../context-actions';
+
+export enum MODIFIER {
+  CTRL = 'MODIFIER_CTRL',
+  CMD = 'MODIFIER_CMD',
+  ALT = 'MODIFIER_ALT',
+  OPTION = 'MODIFIER_OPTION',
+  SHIFT = 'MODIFIER_SHIFT',
+}
+
+// The value should match the KeyboardEvent.key value
+// This is the value which will be displayed if no display overrides are done in the OS specific getDisplayString methods
+export enum KEY {
+  A = 'A',
+  B = 'B',
+  C = 'C',
+  D = 'D',
+  E = 'E',
+  F = 'F',
+  G = 'G',
+  H = 'H',
+  I = 'I',
+  J = 'J',
+  K = 'K',
+  L = 'L',
+  M = 'M',
+  N = 'N',
+  O = 'O',
+  P = 'P',
+  Q = 'Q',
+  R = 'R',
+  S = 'S',
+  T = 'T',
+  U = 'U',
+  V = 'V',
+  W = 'W',
+  X = 'X',
+  Y = 'Y',
+  Z = 'Z',
+  BACKSPACE = 'Backspace',
+  ESCAPE = 'Escape',
+  ENTER = 'Enter',
+  DELETE = 'Delete',
+  SLASH = '/',
```

Comment:
What about the number keys (0-9), backtick/tilde, -/=, comma, period, colon, semicolon, backslash, F keys (eg. `F1`, etc), PageUp/PageDown, Home/End, NumLock, ScrollLock, Pause

Reply:
Yes to function keys, and character keys, no to the others.